src/style.scss

```diff
@@ -1,3 +1,49 @@
-.App {
-    background: green;
+$base-color: #FFCD1B;
+$black-color: #202020;
+$gray-color: #606060;
+$background-color: #e5e5e5;
+$white-color: #ffffff;
+
+
+.button {
```

Comment:
According to ticket [RD-3](https://trello.com/c/85fYF7em/3-configure-scss-modules), you should be using SCSS Modules in your project instead of global stylesheet.

Example:
1. Create a `Button.module.scss` file in the same directory the `Button.js` file.
2. Move the `.button` class to the new file
3. Import your SCSS Module in your Button component:
```jsx
import styles from './Button.module.scss';
```
4. Use your classes in JSX this way:
```jsx
<button
    className={styles.button}
    ...
>...</button>
```